pkgs/development/python-modules/fs/default.nix

```diff
@@ -12,6 +12,8 @@
   pytz,
   setuptools,
   six,
+  isPyPy,
+  glibcLocales,
```

Comment:
Keep them alphabetically ordered.

Reply:
I mean `lib` & `stdenv` are not part of the sorted block, so i did not noticed it tbf